zerver/lib/bugdown/__init__.py

```diff
@@ -43,6 +43,7 @@
 import six
 from six.moves import range, html_parser
 from six import text_type
```

Comment:
I see code usage, but I guess it should be changed (together with the code part - try it.